src/components/Map/USACountyMap.js

```diff
@@ -91,45 +88,41 @@ const USACountyMap = React.memo(
               <Geographies geography={geoStates}>
                 {({ geographies }) =>
                   geographies
-                    .filter(geo => isStateFips(geo.id))
+                    .filter(geo => stateFipsCodes.includes(geo.id))
                     .map(geo => {
-                      const { name } = geo.properties;
                       const fipsCode = geo.id;
-                      const stateCode = getStateCode(name);
-                      const stateUrl = `/${getCanonicalUrl(fipsCode)}`;
```

Comment:
I think you can remove `getCanonicalUrl` + a couple other functions that `getCanonicalUrl` uses now! (I only say this because I have a very similar change stashed away somewhere :) )

Reply:
will do the cleanup on a separate PR